paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetActivity.kt

```diff
@@ -59,15 +61,17 @@ internal class CustomerSheetActivity : AppCompatActivity() {
                             is CustomerSheetViewState.SelectPaymentMethod -> {
                                 CustomerSheetScreen(
                                     header = currentViewState.title,
-                                    isLiveMode = false,
-                                    isProcessing = false,
-                                    isEditing = false,
-                                    onBackPressed = {
-                                        onBackPressedDispatcher.onBackPressed()
-                                    },
-                                    onEdit = {
-                                        TODO()
-                                    }
+                                    paymentMethods = currentViewState.paymentMethods,
+                                    selectedPaymentMethodId =
```

Comment:
It feels like `CustomerSheetScreen` should just take in the view state as a parameter.